expenses/migrations/0002_alter_expenses_date.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.0.3 on 2022-04-11 15:25
+
+import datetime
+from django.db import migrations, models
+from django.utils.timezone import utc
```

Comment:
I don't see that you have changed something in the expenses model, why is there a new migration file?

Reply:
It seems like it's an issue with the expenses model date field. I will create a PR to fix it

Reply:
Great work!